content/docs/1.13/apis.md

```diff
@@ -0,0 +1,81 @@
+---
+title: APIs
+hasparent: true
+---
+
+Jaeger components implement various APIs for saving or retrieving trace data.
+
+TODO
+
+  * stable
+  * restricted
+  * deprecated
+
+## Span reporting APIs
+
+Agent and Collector are the two components of the Jaeger backend that can receive spans and forward them to storage.
+
+### Thrift over UDP (stable)
+
+The Agent can only receive spans over UDP in Thrift format. The primary API is a UDP packet that contains a Thrift-encoded `Batch` struct defined in [jaeger.thrift][jaeger.thrift] IDL file, located in the [jaeger-idl][jaeger-idl] repository. Most Jaeger Clients use Thrift's `compact` encoding, however some client libraries do not support it (notably, Node.js) and use Thrift's `binary` encoding (sent to  a different UDP port). The Agent's API is defined by [agent.thrift][agent.thrift] IDL file.
+
+For legacy reasons, the Agent also accepts spans in Zipkin format, however, only very old versions of Jaeger clients can send data in that format and it is officially deprecated.
+
+### Protobuf via gRPC (stable)
+
+In a typical Jaeger deployment, Agents receive spans from Clients and forward them to Collectors. Since Jaeger version 1.11 the official and recommended protocol between Agents and Collectors is gRPC with Protobuf. The Protobuf IDL [collector.proto][collector.proto] is currently located in the main Jaeger repository, under [model/proto/api_v2][collector.proto]. In the future it will be moved to [jaeger-idl][jaeger-idl] repository ([jaeger-idl/issues/1234](TODO-link)).
```

Comment:
Just wondering whether the first (and possibly second) sentence is required - do we need to provide context for the particular API?

Don't we just need to explain what the API is?

Reply:
I think a bit of context doesn't hurt, because otherwise people will think "why there are so many reporting APIs, which one should I use".

I think  the 3rd sentence should start a new paragraph though.